playground/static/main.js

```diff
@@ -265,22 +271,43 @@ async function updateOutput(input) {
                 errors.forEach(addError);
                 warnings.forEach(addWarning);
 
+                overleafButton.style.display = "none";
```

Comment:
Maybe remove this line altogether, since we probably don't need two duplicate statements in the same block